### PR TITLE
Make red x green!

### PR DESCRIPTION
**Pull request by szepeviktor**
+1 PHP is better before Composer :)

### CODE REVIEW

**Comment by szepeviktor** [comment]
@loganhenson Half way there!! 🎉

**Comment by szepeviktor** [comment]
@loganhenson Done.

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -10,46 +10,44 @@ jobs:
   test:
     if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
 
-    runs-on: ${{ matrix.os }}
-
-    continue-on-error: ${{ matrix.experimental }}
-
     strategy:
-      fail-fast: true
```
Comment (szepeviktor) [inline .github/workflows/run-tests.yml:18]: You may keep fail-fast ...

**Comment by szepeviktor** [comment]
Fixed.

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -31,9 +35,12 @@ jobs:
           php-version: ${{ matrix.php }}
           extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlit3, pdo_sqlite
```
Comment (ramsey) [inline .github/workflows/run-tests.yml:36]: `sqlite3` is misspelled here, which is why one of the errors is showing up in the logs

**Review by ramsey — COMMENTED** [review]
I noticed that there's an error with Composer in the PHP 8 build:

    The process '/usr/local/bin/composer' failed with exit code 2

I looked at the raw logs, and they didn't help. If you keep seeing this, please let me know. I'll update ramsey/composer-install@v1 with better logging and keep an eye out for builds on this repo to see if this causes problems.

File: .github/workflows/run-tests.yml (hunk 2 above)
Comment (szepeviktor) [reply .github/workflows/run-tests.yml]: Thanks.

**Comment by szepeviktor** [comment]
> I'll update ramsey/composer-install@v1 with better logging and keep an eye out for builds on this repo to see if this causes problems.

Showing logs is a good idea.

This project has dependency problems on PHP 8

>     - phpspec/prophecy 1.11.1 requires php ^7.2 -> your PHP version (8.0.0-dev) does not satisfy that requirement.

**Comment by ramsey** [comment]
So, I probably need to display all the composer command output, then. That would help. Thanks!

**Comment by szepeviktor** [comment]
@loganhenson Are a bit happy about it?

**Comment by loganhenson** [comment]
@szepeviktor great work, thanks for taking the time to figure that out!

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**